audioGuestBook.service.template

```diff
@@ -6,7 +6,7 @@ After=multi-user.target
 WorkingDirectory=<path-to-project>
 Type=simple
 Restart=always
-ExecStart=/usr/bin/env python3 <path-to-project>/audioGuestBook.py
+ExecStart=/usr/bin/env python3 <path-to-project>/src/audioGuestBook.py
```

Comment:
I don't believe this will use the venv that you setup during install.

Reply:
This is not the best solution, but if you want to force python to use the libraries you installed in a venv in some specific place, you can set the `PYTHONPATH` environment variable.

Like in this example:
https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/biod/study/init/fingerprint_study_virtualenv.conf#23

I think in systemd, the syntax would be `Environment=PYTHONPATH=<some-abosolute-path>`, then the `ExecStart`.

---

Otherwise, I think you can just launch the python file using the python3 bin/launcher in the venv, like in the following example:

Say you have the following `~/requirements.txt` file:
```
tqdm==4.66.2
```

You could setup a root owned venv like this:
```bash
sudo mkdir -p /var/lib/rotary-phone/venv
sudo python3 -m venv /var/lib/rotary-phone/venv
sudo /var/lib/rotary-phone/venv/bin/pip3 install -r ~/requirements.txt
# To manually run the python file, you could do the following:
sudo /var/lib/rotary-phone/venv/bin/python3 <path-to-project>/audioGuestBook.py
```

Then, you would configure systemd service, like the following:

```systemd
# Running as root
ExecStart=/var/lib/rotary-phone/venv/bin/python3 <path-to-project>/audioGuestBook.py
```

Reply:
ditched venv since deploying on a full img basis now, everything is relative to a single superuser